src/compiler/file_watcher/file_watcher.ts

```diff
@@ -1,20 +1,35 @@
-import { Actor } from '@dfinity/agent';
+import { Actor, ActorMethod, ActorSubclass } from '@dfinity/agent';
 import { watch } from 'chokidar';
-import { readFileSync, writeFileSync } from 'fs';
+import { writeFileSync } from 'fs';
 
 import { createAuthenticatedAgent, whoami } from '../../../dfx';
 import { getCanisterJavaScript } from '../get_canister_javascript';
 import { ok } from '../utils/result';
 
+type ActorReloadJs = ActorSubclass<_SERVICE>;
```

Comment:
Authentication here needs to be fixed

Reply:
This will be addressed before the next release, written down in this issue: https://github.com/demergent-labs/azle/issues/1762